data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1011,84 +1011,129 @@ mission "Hai Reveal [B13] Freedom"
-			`	Having worked through the spaceport and down the access tunnel to the settlement, the vast blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side though, and has not been sealed very well against being reopened from this side. Before long it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`
+			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out wearing mechanized battle suits with armor-plated exoskeletons and weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. With pirates focusing their fire on the imposing suits, the Unfettered make many attacks of opportunity, inflicting massive casualties. The pirates outnumber the attackers ten to one, but this huge advantage in equipment, training, and discipline, gives the attackers an overwhelming advantage.`
+			`	The squads move room to room, neutralizing all opposition with ease until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten seconds. Here some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and swords. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`
+			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many are missing in action because the pirates were very good at trapping the suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate mechs, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies after a pirates used miner's cutting power-arm extensions to great effect.`
```

Comment:
```suggestion
			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many are missing in action because the pirates were very good at trapping the suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate mechs, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies, after the pirates used a miner's cutting power-arm extensions to great effect.`
```